packages/config/config/devices/0x031e/lzw41.json

```
@@ -0,0 +1,76 @@
+// Inovelli LZW41
+// Tunable White Smart Bulb
+{
+	"manufacturer": "Inovelli",
+	"manufacturerId": "0x031e",
+	"label": "LZW41",
+	"description": "Tunable White Smart Bulb",
+	"devices": [
+		{
+			"productType": "0x0006",
+			"productId": "0x0001",
+			"zwaveAllianceId": 3614
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "State After Power Outage",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Prior state",
+					"value": 0
+				},
+				{
+					"label": "Force on",
+					"value": 1
+				},
+				{
+					"label": "Force off",
+					"value": 2
+				}
+			]
+		},
+		"51": {
+			"label": "Color Adjustment (Warm White)",
+			"description": "Changes the kelvin from 2700k (1387) to 4999k (4999)",
+			"valueSize": 2,
+			"minValue": 1387,
+			"maxValue": 4999,
+			"defaultValue": 1387,
```

This seems off. 0x1387 == 4999 (dec), so that was likely a mixup.
```suggestion
			"valueSize": 2,
			"minValue": 2700,
			"maxValue": 4999,
			"defaultValue": 2700,
```

---

I went back to the manual. The range is 2700-4999, default of 2700.